token/services/identity/driver/config.go

```diff
@@ -6,15 +6,23 @@ SPDX-License-Identifier: Apache-2.0
 
 package driver
 
-import (
-	"github.com/hyperledger-labs/fabric-token-sdk/token/driver"
-	"github.com/hyperledger-labs/fabric-token-sdk/token/services/identity/config"
-)
+type ConfiguredIdentity struct {
+	ID        string      `yaml:"id"`
+	Default   bool        `yaml:"default,omitempty"`
+	Path      string      `yaml:"path"`
+	CacheSize int         `yaml:"cacheSize"`
+	Type      string      `yaml:"type,omitempty"`
```

Comment:
Is this a RoleType or an IdentityType? We could type this.

Reply:
the name of struct tells